graphql/schema.graphql

```diff
@@ -11,6 +11,7 @@ type Query {
     sortedBy: String = "created"
   ): OrganizationRequestResult
     @cacheControl(scope: PRIVATE, maxAge: SHORT)
+    @validateAdminUserAccess(orgPermission: "buyer_organization_view")
```

Comment:
for now lets not add the directive to APIs that don't have it. Lets just add the parameter to APIs that already have either `validateAdminUserAccess` or `validateStoreUserAccess`.

Reply:
but if we don't add in getOrganizations for example the permissions will pass. Will it really be like that?